06-unittest.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: page
+title: Testing
+subtitle: The Unittest Module
+minutes: 10
+---
+> ## Learning Objectives {.objectives}
+>
+> -   Understand how to use Unittest to write and run test cases
+> -   Understand that Unittest can be used for many styles of test, not just unit tests
+> -   
+> -   
+
+The Python standard library provides a module for writing and running tests, called `unittest`, which is documented for [Python3.6](https://docs.python.org/3.6/library/unittest.html?highlight=unittest#module-unittest) and [Python2.7](https://docs.python.org/2/library/unittest.html?highlight=unittest#module-unittest).
+
+This module is useful for writing all styles of test, including unit tests and functional tests.
```

Comment:
Since we already cover pytest, perhaps this would be a good place to justify the need to learn both (or when it is or is not necessary to learn both). My general feeling is that pytest is much more friendly to beginners.